STYLE_GUIDE.md

```diff
@@ -0,0 +1,76 @@
+# Introduction
+
+This is the code style guide for the FAST tooling repository. When contributing to this project please follow the standards defined in this guide.
+
+## Files
+
+File and folder naming in general should not include special characters or space. Alphanumeric is preffered, separated by dash or underscore depending on the type of file. Names should be as concise as possible while also being meaningful.
```

Comment:
Unsure if it's necessary but do you want to mention anything about *how* we name things?

Reply:
Added a blurb at the end about following guidance as set by a standards governing body. I think that's as close as we can get and then evaluate PRs on case-by-case basis.